lib/ansiblelint/__main__.py

```diff
@@ -23,17 +23,19 @@
 import errno
 import logging
 import pathlib
+import os
 import sys
 
 import ansiblelint.formatters as formatters
-from ansiblelint import cli, default_rulesdir, RulesCollection, Runner
+from ansiblelint import cli, RulesCollection, Runner
 from ansiblelint.utils import get_playbooks_and_roles
 from ansiblelint.utils import normpath, initialize_logger
 from ansiblelint.generate_docs import rules_as_rst
 from typing import Any, Set
 
 
 _logger = logging.getLogger(__name__)
+default_rulesdir = os.path.join(os.path.dirname(__file__), 'rules')
```

Comment:
shouldn't this live somewhere in utils?

Reply:
As someone else said it before, presence of "utils" is another anti-pattern as that is likely to become the kitchensink of the package.

Default rules directory is specific only to the command line, so it makes sense to use either `__main__` or `cli` modules.

Take a look at https://hackmd.io/@42/ansible-lint-structure -- i build it in order to help us decide where to move things.

Reply:
So put it in CLI then. This way you'd not have to change the help strings.

Reply:
If you want I can even add _ prefix to it, its only used internally.

Reply:
yeah, that'd be fine